### DISCUSSION TITLE
icon pack in miniPaint?

### DISCUSSION THREAD

**Discussion by ghost** [viliusle/miniPaint#352] [Q&A] [answered]
Hi everyone.

Have you all thought about adding browser icon pack support? for people to create screen prototypes in minipaint?

*idea*
![image](https://user-images.githubusercontent.com/111701513/216078567-ccc5712a-4690-4717-b3e8-e8a48acf6bf7.png)

*idea*
![image](https://user-images.githubusercontent.com/111701513/215929390-6ad962c4-2f93-4d10-8950-739ed0741daf.png)

*what do you all think of this idea?*

**Comment by viliusle** [comment, marked as answer]
Tools > sprites should do similar thing, it has some issues and I will work on it. Basically user opens lots of small icons, generates sprites and export single image with lots of icons on it.
